packages/bot-skeleton/src/scratch/hooks/block_svg.js

```diff
@@ -269,7 +269,19 @@ Blockly.BlockSvg.prototype.setCollapsed = function (collapsed) {
             this.setCollapsed(false)
         );
 
-        this.appendDummyInput(COLLAPSED_INPUT_NAME).appendField(field_label).appendField(field_expand_icon).init();
+        if (this.type === 'procedures_defreturn' || this.type === 'procedures_defnoreturn') {
+            const funcLocale = this.getInput('').fieldRow[0].text_;
+            const funcName = this.getFieldValue('NAME');
+            const args = ` (${this.arguments.join(', ')})`;
+
+            this.appendDummyInput(COLLAPSED_INPUT_NAME)
+                .appendField(new Blockly.FieldLabel(funcLocale, field_class))
+                .appendField(new Blockly.FieldLabel(funcName + args, 'header__title'))
```

Comment:
`header__title` is the most suitable class i've found among existing, but it is a bit redundant.
```
.header__title {
    display: inline-block;
    font-weight: bold;
    color: var(--text-prominent);
}
```

Reply:
I suggest you use a more specific class name e.g workspace__collapse__header-title

Reply:
this is an existing one, i haven't modified the css. So you suggest to add a new class to css, like `.workspace__collapes__header-title` or `function-block__collapsed__label`? But is it ok to have a class like that for the only rule of `font-weight: bold;`, maybe it is better to create a class `.font-bold` to reuse it where necessarily? And BTW can you please tell where the appropriate css file is located?